Solutions/Cyborg Security HUNTER/Hunting Queries/Excessive Windows Discovery and Execution Processes - Potential Malware Installation.yaml

```diff
@@ -0,0 +1,57 @@
+id: 00907853-c08c-49ad-b92e-b1df491b843f
+name: Excessive Windows Discovery and Execution Processes - Potential Malware Installation
+description: |
+  'This package utilizes a list of commonly abused LOLB which an attacker or malware would execute in quick succession. The presence of multiple executions of the programs within the list can be indicative of an infection or malicious activity occurring on a victim host. To reduce false positives, distinct counts per process name can be utilized to ensure over 5 unique processes from the list were executed versus just checking more than 6 events were generated on the host.'
```

Comment:
Hi, 
Please reduce the description to 255 characters for all the Hunting Queries.